vignettes/input_preparation_workflow.Rmd

```diff
@@ -108,7 +108,8 @@ evidence_proteotypic <- evidence %>%
     string = proteins,
     pattern = ";",
     negate = TRUE
-  ))# adds new column with logicals that are TRUE if the peptide can be assigned to only one protein and FALSE if it can be assigned to multiple
+  )) %>%# adds new column with logicals that are TRUE if the peptide can be assigned to only one protein and FALSE if it can be assigned to multiple
+  mutate(is_contaminant = ifelse(potential_contaminant == "+", TRUE, FALSE)) # adds new column with logicals indicating if peptide is coming from a potential contaminant
```

Comment:
you could just overwrite the existing potential_contaminant column with this information instead of making a new one.

Reply:
I think it is easier if I leave it there so that people can see if the original input fits the output of the mutate() function here.